templates/desktop/education.html

```diff
@@ -1,151 +1,137 @@
-            </li>
-
-            <li class="grid-list__item four-col">
-                <div class="grid-list__icon one-col">
-                    <img class="grid-list__img" src="{{ ASSET_SERVER_URL }}ff1a448c-marble-icon.png" alt="Marble icon" />
-                </div>
-                <div class="grid-list__description three-col last-col">
-                    <h3>Marble</h3>
-                    <p>Extensive virtual world atlas for exploring the earth and other planets.</p>
-                </div>
-            </li>
-
-            <li class="grid-list__item four-col last-col">
-                <div class="grid-list__icon one-col">
-                    <img class="grid-list__img" src="{{ ASSET_SERVER_URL }}476dda79-italc-icon.png" alt="iTalc icon" />
-                </div>
-                <div class="grid-list__description three-col last-col">
-                    <h3>iTalc</h3>
-                    <p>IT tool which allows teachers to manage individual computers within school’s network.</p>
-                </div>
-            </li>
-
-            <li class="grid-list__item four-col last-row">
-                <div class="grid-list__icon one-col">
-                    <img class="grid-list__img" src="{{ ASSET_SERVER_URL }}61c45d14-logo-stellarium.svg" alt="Stellarium icon" />
-                </div>
-                <div class="grid-list__description three-col last-col">
-                    <h3>Stellarium</h3>
-                    <p>Planetarium software that displays 3D realistic skies in real time.</p>
-                </div>
-            </li>
-
-            <li class="grid-list__item four-col last-row">
-                <div class="grid-list__icon one-col">
-                    <img class="grid-list__img" src="{{ ASSET_SERVER_URL }}916e80c1-logo-scratchcat.svg" alt="Scratch icon" />
-                </div>
-                <div class="grid-list__description three-col last-col">
-                    <h3>Scratch</h3>
-                    <p>Collaborative programming environment for creating interactive stories, games and animation.</p>
-                </div>
-            </li>
-
-            <li class="grid-list__item four-col last-row last-col">
-                <div class="grid-list__icon one-col">
-                    <img class="grid-list__img" src="{{ ASSET_SERVER_URL }}8bd49f5f-logo-biogenesis.jpeg" alt="Biogenesis icon" />
-                </div>
-                <div class="grid-list__description three-col last-col">
-                    <h3>Biogenesis</h3>
-                    <p>An artificial life program that simulates the processes involved in the evolution of organisms.</p>
-                </div>
-            </li>
-        </ul>
+  </div>
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h3>Penn Manor school district, USA</h3>
+      <p style="font-size: 1.5rem; line-height: 1.15;">Penn Manor saved $345,000 on 1,725 Ubuntu PCs at a fraction of what proprietary software costs.</p>
```

Comment:
Avoid inline styles. This looks like it could use one of the [heading classes](https://docs.vanillaframework.io/en/base/typography#heading-classes).